eng/versioning/version_client.txt

```diff
@@ -34,5 +34,5 @@ com.azure:azure-storage-queue;12.2.0-beta.1;12.2.0-beta.2
 # version. Unreleased dependencies are only valid for dependency versions
 # Format;
 # unreleased_<groupId>:<artifactId>;dependency-version
-unreleased_com.azure:azure-core;1.2.0-beta.1
+unreleased_com.azure:azure-core;1.2.0
```

Comment:
Once we release azure-core 1.2.0 this entry should go away and the dependency version should be changed above to 1.2.0.